examples/serverless/serverless-middy/src/handler.ts

```diff
@@ -25,5 +24,23 @@ export const handler = async (
   };
 };
 
-export const middyHandler = middy(handler)
-  .use(jsonBodyParser());
+// Example with official middleware, you need to do a POST request in order to send the body
+export const getHandler = async (
+  event: APIGatewayEventMiddyNormalised
+): Promise<APIGatewayProxyResult> => {
+  return {
+    statusCode: 200,
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify("Your are in the safe list"),
+  };
+};
+
+export const officialMiddlewareHandler = middy(postHandler).use(
+  jsonBodyParser()
+);
+
+export const customMiddlewareHandler = middy(getHandler).use(
+  customMiddleware()
+);
```

Comment:
we can have a dir called `src/hanlers` and have two files in there `get.ts` and `post.ts`